sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/FullBackupResponse.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.keyvault.administration.implementation.models;
+
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.rest.ResponseBase;
+
+/** Contains all response data for the fullBackup operation. */
+public final class FullBackupResponse extends ResponseBase<FullBackupHeaders, FullBackupOperation> {
```

Comment:
Why is this public? See https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs for what should be public (idiomatic details will differ, of course, but models and clients should largely be the same).

Reply:
This is auto-generated code and is in an internal package. You can refer to my comments on similar classes on [this PR](https://github.com/Azure/azure-sdk-for-java/pull/12405).